test/functional/change_stream.test.js

```diff
@@ -2609,34 +2619,24 @@ describe('Change Streams', function() {
   describe('tryNext', function() {
```

Comment:
I oddly started getting an error in the `tryNext` tests:

`MongoError: cannot open $changeStream for non-existent database: test`

I thought it would be more robust to use a specifically named database and drop it after each test, to ensure weird behavior doesn't carry over between tests. To that purpose I created the `withTempDb` helper that wraps `withClient` to create and drop the specified database. This refactor made above error go away.